spring.core/src/main/java/com/epam/springcore/task/service/TraineeService.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.springcore.task.service;
+
+import com.epam.springcore.task.model.Trainee;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface TraineeService {
+
+    Optional<Trainee> create (Trainee trainee);
```

Comment:
create (Trainee trainee) methods definition should not contain space between method name and parenthesis